tests/test_parse_tanner_response.py

```diff
@@ -0,0 +1,113 @@
+import unittest
+from unittest.mock import Mock
+from unittest.mock import call
+import asyncio
+import argparse
+import aiohttp
+import shutil
+import os
+import json
+import yarl
+from aiohttp.protocol import HttpVersion
+from utils.asyncmock import AsyncMock
+from snare import HttpRequestHandler
+from utils.page_path_generator import generate_unique_path
+from urllib.parse import unquote
+
+
+class TestParseTannerResponse(unittest.TestCase):
+    def setUp(self):
+        run_args = argparse.ArgumentParser()
+        run_args.add_argument("--tanner")
+        run_args.add_argument("--page-dir")
+        self.main_page_path = generate_unique_path()
+        os.makedirs(self.main_page_path)
+        self.page_dir = self.main_page_path.rsplit('/')[-1]
+        self.meta_content = {"/index.html": {"hash": "hash_name", "content_type": "text/html"}}
+        self.page_content = "<html><body></body></html>"
+        self.content_type = "text/html"
+        with open(os.path.join(self.main_page_path, "hash_name"), 'w') as f:
+            f.write(self.page_content)
+        with open(os.path.join(self.main_page_path, "meta.json"), 'w') as f:
+            json.dump(self.meta_content, f)
+        self.args = run_args.parse_args(['--page-dir', self.page_dir])
+        self.requested_name = '/'
+        self.loop = asyncio.new_event_loop()
+        self.handler = HttpRequestHandler(self.meta_content, self.args)
+        self.handler.run_args.index_page = '/index.html'
+        self.handler.handle_html_content = AsyncMock(return_value=self.page_content)
+
+    def test_parse_type_one(self):
+        self.detection = {"type": 1}
+        self.call_content = b'<html><body></body></html>'
+        self.expected_content = self.page_content
+
+        async def test():
+            (self.res1, self.res2,
```

Comment:
you don't need `self` for result variables here, you will not reuse these variables. So for the future - it's better to avoid such things